### PR TITLE
Update pylint comments and docstrings

### PR DESCRIPTION
**Pull request by insacuri**
### What is the context of this PR?
https://trello.com/c/1iaM7SLX/2531-add-docstrings-and-remove-ignore-pylint-comments

Whilst there was some time, I went through and added some docstrings to public methods and removed a pylint ignore comment that was no longer relevant.

### How to review 
All tests should pass.  Only code changes was a very minor refactor to simplify an already simple function.  Maybe complete a survey to verify that it's fine?

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by bitdivision — APPROVED** [review]

File: app/data_model/answer_store.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def matches_dict(self, answer_dict):
         Check to see if a dict describes an answer the same as this object.
 
         :param answer_dict: A dict providing
```
Comment (MebinAbraham) [inline app/data_model/answer_store.py]: I know its existing but `:param answer_dict: A dict providing`, that doesn't make sense?

File: app/libs/utils.py [hunk 2]
```diff
@@ -2,12 +2,20 @@
...
+    :param tx_id: tx_id to be converted
+    :return: String in the form of xxxx-xxxx-xxxx-xxxx
+    """
     return (tx_id[:4] + '-' + tx_id[4:])[:19]
 
 
 def convert_tx_id_for_boxes(tx_id):
-    # converts the guid tx_id to string of 16 characters with a space between every 4 characters
+    """
+    converts the guid tx_id to string of 16 characters with a space between every 4 characters
```
Comment (MebinAbraham) [inline app/libs/utils.py]: Capitalize?

**Review by MebinAbraham — APPROVED** [review]

File: app/data_model/answer_store.py (hunk 1 above)
Comment (insacuri) [reply app/data_model/answer_store.py]: What would you change it to?  I'm not sure how to describe it sorry

Comment (MebinAbraham) [reply app/data_model/answer_store.py]: A dictionary representation of the answer?

**Pull request merged**